packaging/win32/installer.nsi

```diff
@@ -122,13 +122,11 @@ Function .onInit
     /SD IDOK IDOK uninst
     Abort
 
-    ;Run the uninstaller
+    ;Run the uninstaller sequentially and silently
+    ;https://nsis.sourceforge.io/Docs/Chapter3.html#installerusageuninstaller
     uninst:
       ClearErrors
-      IfSilent +3
-      Exec $R0
-      Goto +2
-      Exec "$R0 /S"
+      ExecWait '"$R0" /S _?=$INSTDIR'
```

Comment:
A sarcastic comment is lacking here :trollface: 

More seriously, what is displayed to the user while the uninstall is done in the background ? doesn't it look like the installer is frozen ? (especially given uninstalling parsec is quiet slow...)

Reply:
Actually the uninstallation is only slow because of the GUI. On my (quite slow) VM, it only takes a couple of seconds. During this period, nothing is show two the user: the dialog is gone, and later the installer pops up.

Reply:
so it's so slow because it displays useless information that scrolls too fast for the user to read them anyway ? O_o

Reply:
Exactly :D